sdk/communication/AzureCommunicationCommon/Source/Identifiers.swift

```diff
@@ -114,31 +138,65 @@ import Foundation
         rawId: String? = nil,
         cloudEnvironment: CommunicationCloudEnvironment = .Public
     ) {
-        self.rawId = rawId
```

Comment:
I'm not entirely sure what you mean here? Do you have an example of how you think it should be implemented? 
I can set up a convenience init but I am not sure how it will work in Objective-C, if it will work at all. I would need to test it out

Reply:
I was thinking something like:

```swift
convenience init(userId: String, isAnonymous: Bool) {
     self.init(userId, isAnonymous, nil, .Public)
}

init(
        userId: String,
        isAnonymous: Bool = false,
        rawId: String? = nil,
        cloudEnvironment: CommunicationCloudEnvironment = .Public
    ) {
  /* set all properties, do the rawId serialization*/
}
```

Reply:
>but I am not sure how it will work in Objective-C


seems to be possible
https://stackoverflow.com/a/36570905/1332034